co-authors-plus.php

```diff
@@ -1070,7 +1070,7 @@ public function ajax_suggest() {
 		$authors = $this->search_authors( $search, $ignore );
 
 		foreach ( $authors as $author ) {
-			echo $author->ID . ' | ' . $author->user_login . ' | ' . $author->display_name . ' | ' . $author->user_email . ' | ' . $author->user_nicename . "\n";
+			echo esc_html( $author->ID . ' | ' . $author->user_login . ' | ' . $author->display_name . ' | ' . $author->user_email . ' | ' . $author->user_nicename . "\n" );
```

Comment:
Might be better to escape the individual elements rather than the entire string (just in case the delimiters/newline end up modified somehow)?